core/src/jsonpp.hpp

```diff
@@ -113,13 +114,17 @@ json & json::operator << (json::_context_t ctxt) throw()
     return *this;
 }
 
-template<typename C>
 inline
-json & operator << (json & j, std::basic_string<C> const & s) throw() { return j << json::string(convert<C,char>(s).c_str()); }
+json & operator << (json & j, std::string const & s) throw() { return j << json::string(s.c_str()); }
 
-template<typename C>
```

Comment:
i think the `C` here matched far too widely.  ANd some of the matches "crashed" with a static assert.